jobs/jobsv1/repository.go

```diff
@@ -25,8 +26,8 @@ func getKey(did identity.DID, id jobs.JobID) ([]byte, error) {
 	if jobs.JobIDEqual(jobs.NilJobID(), id) {
 		return nil, errors.New("job ID is not valid")
 	}
-
-	return append(did[:], id.Bytes()...), nil
+	hexKey := hexutil.Encode(append(did[:], id.Bytes()...))
```

Comment:
This was missing, as it came right after we were indexing with Hex keys. I will take advantage of this situation and add a migration to do this conversion for existing data.

Reply:
I might take this change out into its own PR with the migration that comes with it.